Makefile

```diff
@@ -129,6 +129,9 @@ terraform-app-destroy-review: terraform-app-init ## make passcode=MyPasscode CON
 		$(if $(CONFIRM_DESTROY), , $(error Can only run with CONFIRM_DESTROY))
 		cd terraform/app && terraform destroy -var-file ../workspace-variables/review.tfvars -auto-approve
 
+terraform-app-destroy: terraform-app-init ## make qa destroy passcode=MyPasscode
```

Comment:
This is useful but dangerous. Shall we add a safeguard like above?
`$(if $(CONFIRM_DESTROY), , $(error Can only run with CONFIRM_DESTROY))`

Reply:
Even better, make terraform-app-destroy work for review as well

Reply:
....Correct, which is why i didnt put `-auto-approve`

Reply:
Explained IRL